ui/perfherder/alerts/AlertTableRow.jsx

```diff
@@ -193,6 +194,43 @@ export default class AlertTableRow extends React.Component {
     );
   };
 
+  showTags = (tags) => {
+    return tags.map((item) => (
+      <Badge color="light" key={`${item}`}>
+        {item}
+      </Badge>
+    ));
+  };
+
+  getTags = (alert) => {
+    const { displayTags } = this.state;
+    const { tags } = alert.series_signature;
+
+    if (tags.length && tags[0] !== '') {
+      return (
+        <React.Fragment>
+          {this.showTags(tags.slice(0, 2))}
```

Comment:
The value for the number of tags to be displayed by default (in this case `2`) should be extracted above (it appears in a lot of places)